WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Grouped Products/ProductListSelectorViewController.swift

```diff
@@ -71,7 +71,12 @@ private extension ProductListSelectorViewController {
         let searchViewController = SearchViewController(storeID: siteID,
                                                         command: searchProductsCommand,
                                                         cellType: ProductsTabProductTableViewCell.self)
+        // Disables interactive dismiss action so that we can prompt the discard changes alert.
+        if #available(iOS 13.0, *) {
```

Comment:
With the previous change, I think this won't be necessary.

Reply:
yup! removed in https://github.com/woocommerce/woocommerce-ios/pull/2976/commits/2ad8c7d871dabfe25364030026e3159f250473de